ServiceDocumentSampleComponents/force-app/main/default/classes/FireCreateServiceDocumentInvocableAction.cls

```diff
@@ -0,0 +1,64 @@
+public with sharing class FireCreateServiceDocumentInvocableAction {
```

Comment:
If we're adding these files to ServiceDocumentSampleComponents, we may want to rename the folder instead to ServiceDocumentSamples considering an apex trigger is not a component. Additionally, we should write something in the folder's readme to document that this trigger is separate from the components